apps/web/lib/features/manual-time/add-manual-time-modal.tsx

```diff
@@ -47,40 +51,30 @@ export function AddManualTimeModal(props: IAddManualTimeModalProps) {
 		(e: FormEvent<HTMLFormElement>) => {
 			e.preventDefault();
 
-			const timeObject = {
-				date,
-				isBillable,
-				startTime,
-				endTime,
-				teamId,
+			const startedAt = new Date(date);
+			const stoppedAt = new Date(date);
+
+			// Set time for the started date
+			startedAt.setHours(parseInt(startTime.split(':')[0]));
+			startedAt.setMinutes(parseInt(startTime.split(':')[1]));
+
+			// Set time for the stopped date
+			stoppedAt.setHours(parseInt(endTime.split(':')[0]));
+			stoppedAt.setMinutes(parseInt(endTime.split(':')[1]));
+
+			const requestData: Omit<IAddManualTimeRequest, 'tenantId' | 'employeeId' | 'logType' | 'source'> = {
+				startedAt,
+				stoppedAt,
 				taskId,
 				description,
 				reason,
-				timeDifference
+				isBillable,
+				organizationId: team?.organizationId
 			};
 
-			if (date && startTime && endTime && teamId && taskId) {
```

Comment:
We had condition logic @CREDO23 , "if (date && startTime && endTime && teamId && taskId) {" why you removed it!?

Reply:
I just added it! @evereq